<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Calculator">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>340</height>
      </size>
     </property>
     <attribute name="title">
      <string>Calculator</string>
     </attribute>
     <widget class="QLineEdit" name="screen">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>420</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mult">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="seven">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mod">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
     <widget class="QPushButton" name="four">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minus">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dot">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="three">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equal">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="atan">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sqrt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="six">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plus">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="asin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="five">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="div">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closebrace">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="power">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openbrace">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackSpace">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string>⌫</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearAll">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ln">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nine">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="one">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eight">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zero">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="log">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="acos">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="two">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>260</y>
        <width>20</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>x =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="exInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ex">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Graph">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <attribute name="title">
      <string>Graph</string>
     </attribute>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>581</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>585</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>f(x) =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>220</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ymax =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ymin =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Xmax =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Xmin =</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DrawButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>280</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>17</y>
        <width>175</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Xmin">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>57</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Xmax">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>97</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ymin">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>177</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ymax">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>217</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Credit">
     <attribute name="title">
      <string>Credit</string>
     </attribute>
     <widget class="QLabel" name="labelCreditType">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Тип ежемесячных платежей:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCreditSum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Сумма кредита:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCreditDuration">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Срок кредита:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCreditPercentage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Процентная ставка:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCreditPercentage">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>115</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCreditDuration">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCreditSumm">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>35</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="mainCreditFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>400</width>
        <height>200</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="creditAnnuitet">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>130</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>  Аннуитетный</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="creditDifference">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>160</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Дифференцированный</string>
       </property>
      </widget>
      <widget class="QComboBox" name="creditDuration">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>месяцев</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>лет</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>91</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>% годовых</string>
       </property>
      </widget>
      <zorder>creditDuration</zorder>
      <zorder>creditAnnuitet</zorder>
      <zorder>creditDifference</zorder>
      <zorder>label_7</zorder>
     </widget>
     <widget class="QFrame" name="frame4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>149</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>109</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>69</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame1">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="outputFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>400</width>
        <height>120</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>400</width>
         <height>40</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelMnthPay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>190</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Ежемесячный платеж:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="outputPayment">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>201</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="outputTotalSum">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>201</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelTtlPay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>190</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Общая выплата:</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="outputOverPay">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>5</y>
          <width>201</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelOvrPay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>190</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Переплата по кредиту:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>CALCULATE</string>
      </property>
     </widget>
     <zorder>outputFrame</zorder>
     <zorder>frame4</zorder>
     <zorder>frame3</zorder>
     <zorder>frame1</zorder>
     <zorder>frame2</zorder>
     <zorder>mainCreditFrame</zorder>
     <zorder>labelCreditType</zorder>
     <zorder>labelCreditDuration</zorder>
     <zorder>labelCreditPercentage</zorder>
     <zorder>inputCreditPercentage</zorder>
     <zorder>inputCreditDuration</zorder>
     <zorder>inputCreditSumm</zorder>
     <zorder>labelCreditSum</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
